packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -29,31 +30,34 @@ export interface CallCompositeProps extends BaseCompositeProps<CallCompositeIcon
   onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
 
   /**
-   * Flags to enable/disable visual elements of the {@link CallComposite}.
+   * Flags to hide UI elements of the {@link CallComposite}.
    */
-  visualElements?: CallCompositeVisualElements;
+  hiddenElements?: CallCompositeHiddenElements;
 }
 
 /**
- * Optional features of the {@linnk CallComposite}
+ * Optional features of the {@link CallComposite}
  */
-export type CallCompositeVisualElements = {
+export type CallCompositeHiddenElements = CallControlHiddenElements & {
   /**
    * Surface Azure Communication Services backend errors in the UI with {@link @azure/communication-react#ErrorBar}.
+   * Hidden if set to `true`
    *
-   * @defaultValue false
+   * @default false
```

Comment:
This should remain `@defaultValue` I think https://tsdoc.org/pages/tags/defaultvalue/

Reply:
Oops. Will change it all